app/templating/summary/question.py

```diff
@@ -67,11 +80,11 @@ def _build_checkbox_answers(cls, answer, answer_schema):
         multiple_answers = []
         for option in answer_schema['options']:
             if option['value'] in answer:
-                if option['value'] == 'other':
+                if option['value'].lower() == 'other':
```

Comment:
you should probably add the `.lower()` to radio also. this will keep the behaviour consistent